css/css.go

```diff
@@ -664,7 +664,21 @@ func (c *cssMinifier) minifyProperty(prop css.Hash, values []Token) []Token {
 			}
 		}
 		// removing zero offsets in the previous loop might make it eligible for the next loop
-		if len(values) < 3 {
+		if len(values) == 1 || len(values) == 2 {
```

Comment:
Because I'm accessing `values[0]`, I wasn't sure if I can rely on invariant that values is non-empty, so had to do two comparisons either way - `len(values) > 0 && len(values) < 3` or this one. This one seems a bit cleaner / more explicit about values it accepts.